### ISSUE TITLE
webpack-command: “0:0  error  webpack-stylish: <please report unknown message format>”

### ISSUE THREAD

**Issue by PoziWorld** [evcohen/eslint-plugin-jsx-a11y#445]
Hi, 

This is a great plugin. Thank you for your time and work!

However, there seems to be an issue:

Getting the following error message without any details (for example, when `onChange` is used without `onBlur`, the issue is not specified in the output) with webpack-command:
`0:0  error  webpack-stylish: <please report unknown message format>`

Other plugins (`babel` and `react`) work fine – they output full error messages. That's why posting it here.

 
The following snippet causes no warnings or errors:
```javascript
import React, { PureComponent } from 'react';

export default class Field extends PureComponent {
  render() {
    const { handleChange } = this.props;

    return (
      <label>
        <select onBlur={ handleChange }>
          <option />
        </select>
      </label>
    );
  }
}
```

 
The following snippet causes the aforementioned error:
```javascript
import React, { PureComponent } from 'react';

export default class Field extends PureComponent {
  render() {
    const { handleChange } = this.props;

    return (
      <label>
        <select onChange={ handleChange }>
          <option />
        </select>
      </label>
    );
  }
}
```

 
package.json:
```json
  "devDependencies": {
    "babel": "^6.23.0",
    "babel-core": "^6.26.3",
    "babel-eslint": "^8.2.3",
    "babel-loader": "^7.1.4",
    "babel-plugin-transform-class-properties": "^6.24.1",
    "babel-plugin-transform-runtime": "^6.23.0",
    "babel-preset-env": "^1.7.0",
    "babel-preset-react": "^6.24.1",
    "eslint": "^4.19.1",
    "eslint-loader": "^2.0.0",
    "eslint-plugin-babel": "^5.1.0",
    "eslint-plugin-jsx-a11y": "^6.0.3",
    "eslint-plugin-react": "^7.9.1",
    "react": "^16.4.0",
    "react-dom": "^16.4.0",
    "webpack": "^4.10.2",
    "webpack-command": "^0.2.0"
  }
```

 
.eslintrc:
```json
{
  "env": {
    "browser": true,
    "node": true
  },
  "extends": [
    "plugin:jsx-a11y/recommended"
  ],
  "parser": "babel-eslint",
  "parserOptions": {
    "ecmaVersion": 2018,
    "ecmaFeatures": {
      "jsx": true
    },
    "sourceType": "module"
  },
  "plugins": [
    "babel",
    "jsx-a11y",
    "react"
  ],
  "rules": {
    "babel/new-cap": 1,
    "babel/no-invalid-this": 1,
    "babel/semi": 1,
    "babel/no-unused-expressions": 1,
    "jsx-a11y/label-has-for": [
      2,
      {
        "allowChildren": true
      }
    ]
  }
}
```

 
webpack.config.js:
```javascript
  module: {
    rules: [
      {
        enforce: 'pre',
        test: /\.js$/,
        exclude: /node_modules/,
        loader: 'eslint-loader',
      },
      {
        test: /\.js$/,
        exclude: /node_modules/,
        loader: 'babel-loader',
        query: {
          cacheDirectory: true,
          presets: [
            'env',
            'react',
          ],
          plugins: [
            'transform-class-properties',
            'transform-runtime',
          ],
        },
      },
    ],
  },
  stats: 'verbose'
```

 
Do you happen to know how to make it output the actual error?

If not, would you be able to look into it, please?
 
Thank you.

**Comment by ljharb** [comment]
I’m not sure how this is related to this project (an eslint plugin). Perhaps it should be filed on webpack-stylish?

**Comment by DinoSourcesRex** [comment]
@PoziWorld Hey. I just encountered this and it was because I put a flag in the wrong place. Can you show your entire package.json file (I care about the scripts) and tell me which npm command you ran.

My error: `npm run MYSCRIPT --config -p config.js` the fix was to move the `-p` to the right spot.
